third_party/xla/xla/mlir_hlo/tests/BUILD

```diff
@@ -26,14 +26,9 @@ package(
         tags = [
             "nomsan",  # The execution engine doesn't work with msan, see b/248097619.
         ],
-<<<<<<< HEAD
         #Build failing due to the double deps definition. Should be resolved by next weekly thanks to 
         #https://github.com/llvm/llvm-project/commit/dd5797505ebc2dbfdd58927c4f0a11a1256696eb
         #deps = ["@pypi_lit//:pkg"],
-=======
-        deps = ["@pypi_lit//:pkg"],
-        deps = ["@pypi_lit//:pkg"],
->>>>>>> upstream/master
```

Comment:
Not sure why they doubled the line, but I think we can leave local deps now (since llvm/llvm-project#87290 is merged)